### DISCUSSION TITLE
New tool

### DISCUSSION THREAD

**Discussion by ghost** [chylex/Discord-History-Tracker#145] [General]
Continuation of:
https://github.com/chylex/Discord-History-Tracker/issues/141

**Comment by denetii** [comment]
Hey, hope this is the right place to discuss the new desktop app :) 

I tried out the current build, and it's looking good! It does seem like a better solution if you want to maintain a large database. 

I was wondering how feasible it would be to import 'legacy' JSON archives into the new SQLite DB. It would be pretty sweet if you could do a smooth transition from the JSON files to the new format, so that you could load in your previous archives and then continue archiving from the most recent messages you'd previously saved.

**Reply by chylex** [reply]
The problem is you'd lose too much information, especially the way embeds are stored is completely different and not easily transferable between the formats. If you're okay with losing embeds or at least partially reconstructing them, it could be a nice tool separate from the app.
